carta/cpp/plugins/cartaview/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name"       : "cartaview",
+    "version"    : "0.1",
+    "type"       : "lib",
+    "description": "adds libraries needed for running carta in casa",
+    "about"      : "Written by Jeff Taylor",
+    "depends"    : []
+}
```

Comment:
This plugin is basically a pre-requisite set of libraries to be able to load casacore.

Maybe a simpler solution would be to just add these to the casacore-2.0.1 plugin, especially since I noticed it already contains cfitsio and libwcs libraries...